public/docs/features.md

```diff
@@ -11,121 +10,128 @@ If you experience any _issues_, feel free to report it on [**GitHub**](https://g
 Or meet us on [**Matrix.org**](https://riot.im/app/#/room/#codimd:matrix.org) for dev-talk and interactive help.
 **Thank you very much!**
 
-Workspace
-===
-## Modes
-**Desktop & Tablet**
+## Workspace
+
+### Modes
+
+#### Desktop & Tablet
 
 <i class="fa fa-edit fa-fw"></i> Edit: See only the editor.
 <i class="fa fa-eye fa-fw"></i> View: See only the result.
 <i class="fa fa-columns fa-fw"></i> Both: See both in split view.
 
-**Mobile**
+#### Mobile
 
 <i class="fa fa-toggle-on fa-fw"></i> View: See only the result.
 <i class="fa fa-toggle-off fa-fw"></i> Edit: See only the editor.
 
-## Night Mode:
+### Night Mode
+
 When you are tired of a white screen and like a night mode, click on the little moon <i class="fa fa-moon-o"></i> and turn on the night view of CodiMD.
 
 The editor view, which is in night mode by default, can also be toggled between night and day view using the the little sun<i class="fa fa-sun-o fa-fw"></i>.
 
-## Image Upload:
+### Image Upload
+
 You can upload an image simply by clicking on the camera button <i class="fa fa-camera"></i>.
 Alternatively, you can **drag-n-drop** an image into the editor. Even **pasting** images is possible!
-This will automatically upload the image to **[imgur](http://imgur.com)**, **[Amazon S3](https://aws.amazon.com/s3/)**, **[Minio](https://minio.io)** or **local filesystem**, nothing to worry about. :tada:
+This will automatically upload the image to **[imgur](http://imgur.com)**, **[Amazon S3](https://aws.amazon.com/s3/)**, **[Minio](https://minio.io)** or the **local filesystem** (depending on the instance's configuration), nothing to worry about. :tada:
 ![imgur](https://i.imgur.com/9cgQVqD.png)
```

Comment:
please add a blank line before this one, the image currently sticks directly to the paragraph (an error that was in the earlier version too, not your fault!)

Reply:
```suggestion

![imgur](https://i.imgur.com/9cgQVqD.png)
```